Automattic/jetpack
# Social: Update connection toggles to be button switches

Updates the toggles to match designs from p1HpG7-jqQ-p2

<table>
<tr>
 <td><img width="292" alt="image" src="https://github.com/Automattic/jetpack/assets/746152/623ea80c-21fb-4313-98df-41bdb8697686">

 <td>
<img width="285" alt="image" src="https://github.com/Automattic/jetpack/assets/746152/2de6b286-9f03-44b5-9351-05026bb8597d">



<tr>
 <td>Before
 <td>After
</table>


## Proposed changes:

* Updates the toggle components to be image switches
* Moves the toggles before the limits notice

### Other information:


## Jetpack product discussion
p1HpG7-jqQ-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:






* With Social enabled and a few connections active, visit the editor
* Open the Jetpack plugin sidebar
* Confirm the connection buttons look like in the screenshots
* Click the add connection button
* Confirm you're taken to add a new connection
